view/stache/doc/helpers/is.md

```diff
@@ -0,0 +1,27 @@
+@function can.stache.helpers.is {{#is expr1 expr2 expr3}}
```

Comment:
if you look at how the other helpers look in the sidebar of:

http://canjs.com/docs/can.stache.html

You'll notice that `{{#is expr1 expr2 expr3}}` doesn't really fit in.  At least, I think expr3 should be removed: `{{#is expr1 expr2}}`.  We could start showing expanded arguments like `{{#is expr...}}`